src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -58,6 +58,28 @@ function warnIfValueIsNull(props) {
   }
 }
 
+function warnIfDuplicateValues(inst) {
```

Comment:
Should I move it to ReactDOMNullInputValuePropDevtool.js or create a new file? If a new file, what should I name it? Thanks.